pkg/storage/unified/search/bleve.go

```diff
@@ -339,14 +334,19 @@ func (b *bleveIndex) CountRepositoryObjects(ctx context.Context) (map[string]int
 	if err != nil {
 		return nil, err
 	}
-	rsp := make(map[string]int64)
+	vals := make([]*resource.CountRepositoryObjectsResponse_KindCount, 0)
 	f, ok := found.Facets["count"]
 	if ok && f.Terms != nil {
 		for _, v := range f.Terms.Terms() {
-			rsp[v.Term] = int64(v.Count)
+			vals = append(vals, &resource.CountRepositoryObjectsResponse_KindCount{
+				Repository: v.Term,
+				Group:      b.key.Group,
+				Resource:   b.key.Resource,
```

Comment:
Shouldn't this be `kind` instead `resource`? Trying to play more in the conventional k8s way. We would have a bunch of resources with group and kind for each repository.